packages/react-jsx-highcharts-gantt/src/components/Pathfinder/Pathfinder.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { attempt } from 'lodash-es';
+import {getModifiedProps} from 'react-jsx-highcharts';
+
+class Pathfinder extends Component {
+
+  static propTypes = {
+    getChart: PropTypes.func.isRequired, // Provided by ChartProvider
+    enabled: PropTypes.bool.isRequired,
+    dashStyle: PropTypes.string(),
+    lineWidth: PropTypes.number()
+  };
+
+  static defaultProps = {
+    enabled: true
```

Comment:
These default props should mirror the [pathfinder api](https://api.highcharts.com/gantt/pathfinder.dashStyle) defaults

I.e.

```js
static defaultProps = {
  enabled: true,
  dashStyle: 'solid',
  lineWidth:  1,
  type: 'straight|simpleConnect'
}
```